autogen/src/dr.rs

```diff
@@ -291,12 +315,12 @@ pub fn gen_dr_operand_kinds(grammar: &[structs::OperandKind]) -> TokenStream {
                 }
             }
         });
-        let unwraps = kind_and_ty.into_iter().map(|(kind, ty)| {
+        let unwraps = kind_and_ty.iter().map(|(kind, ty)| {
```

Comment:
`kind_and_ty` doesn't appear to be used after this line (ie. might be a development artifact), a move with `into_iter` and without the extra `clone()` below should be faster?